src/settings_type.h

```diff
@@ -316,7 +316,9 @@ struct NetworkSettings {
 	std::vector<std::string> server_authorized_keys; ///< Public keys of clients that are authorized to connect to the game.
 	std::string rcon_password;                            ///< password for rconsole (server side)
 	std::vector<std::string> rcon_authorized_keys; ///< Public keys of clients that are authorized to use the rconsole (server side).
+	bool allow_unsecure_admin_login; ///< Whether to allow logging in as admin using the unsecured old JOIN packet.
```

Comment:
The famous unsecured vs insecured :D

The problem with unsecured, `unsecure` is not a word in the English language. Only `unsecured` is. "Your bike is unsecured". Something like that. Which makes it a really annoying word to deal with.

In general, it is common to call a setting like this `allow_insecure_admin_login`, as you know it is insecure, as the password is unsecured :D Still with me? :P

```suggestion
	bool allow_insecure_admin_login; ///< Whether to allow logging in as admin using the unsecured old JOIN packet.
```

https://grammarhow.com/unsecure-vs-insecure/ has a nice write-up about it.